utilities.js

```diff
@@ -1,24 +1,8 @@
-const products = require('./src/data/products')
-const categories = require('./src/data/categories')
+'use strict';
+
 module.exports = {
-    getProductsByCategoryId : (categoryId) => {
-        return products.products.filter(value => value.categoryId === categoryId)
-    },
-    getCategoryById: (id) => {
-        return categories.categories.filter(value => value.id === id)[0];
-    },
-    getProductById: (id) => {
-        return products.products.filter(value => value.id === id)[0];
-    },
-    sendNotFoundResponse: (res, message) => {
-        res.statusCode = 404
-        res.send(message)
-    },
-    sendInternalServerErrorResponse: (res) => {
-        res.statusCode = 500
-        res.end()
-    },
-    injectCategoryName: (product) => {
-        product['categoryName'] = module.exports.getCategoryById(product.categoryId).categoryName
-    }
-}
+  sendNotFoundResponse: (res, message) => {
+    res.statusCode = 404;
+    res.send(message);
```

Comment:
Don't send and responses from `utilities`. You can prepare an response object here, and you should send it from the API layer